web/src/features/charts/DisabledMessage.tsx

```diff
@@ -0,0 +1,7 @@
+export function DisabledMessage({ message }: { message: JSX.Element | string }) {
```

Comment:
```suggestion
export function DisabledMessage({ message }: { message: JSX.Element }) {
```

Should always return an element right?

Reply:
That is not the return type but the type of `message` . I could change it to only be of type string so we could also keep it as a translation string as @VIKTORVAV99 suggests.